trajopt/src/problem_description.cpp

```diff
@@ -1782,7 +1787,8 @@ void CollisionTermInfo::hatch(TrajOptProb& prob)
                                                        prob.GetVarRow(i, 0, n_dof),
                                                        prob.GetVarRow(i + 1, 0, n_dof),
                                                        expression_evaluator_type,
-                                                       discrete_continuous);
+                                                       discrete_continuous,
+                                                       safety_margin_buffer);
```

Comment:
For costs should we eliminate this parameter  and just pass zero to the Collision evaluator?

Reply:
I made the default value of this parameter zero in the Tesseract CollisionCostConfig in [this PR](https://github.com/ros-industrial-consortium/tesseract/pull/215). There might be some reason to have a non-zero buffer distance for collision costs, so I don't know that we should disable it entirely.